components-compose/src/main/java/uk/gov/hmrc/components/compose/organism/card/IconButtonCardView.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.hmrc.components.compose.organism.card
+
+import androidx.annotation.DrawableRes
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import uk.gov.hmrc.components.compose.atom.button.IconButton
+import uk.gov.hmrc.components.compose.organism.HmrcCardView
+
+@Composable
+fun IconButtonCardView(
+    titleText: String,
+    @DrawableRes iconResId: Int,
```

Comment:
As the same icon resource we are using, can we make default iconResId ->  R.drawable.ic_help_outline,
@DrawableRes iconResId: Int = R.drawable.ic_help_outline

Reply:
I agree with that -> _"for the default icon, Im going to push back on that, because its only the example we’re using ic_help_outline , but it should be up to the consumer app to set the icon, rather then using a default one."_